packages/admin/blocks-admin/src/blocks/factories/createBlocksBlock.tsx

```diff
@@ -570,42 +579,46 @@ export function createBlocksBlock<AdditionalItemFields extends Record<string, un
-                                                            </IconButton>
-                                                            <IconButton
-                                                                onClick={handleCopySelectedBlocks}
-                                                                size="large"
-                                                                disabled={selectedCount === 0}
-                                                            >
-                                                                <Copy />
-                                                            </IconButton>
-                                                            <IconButton onClick={() => pasteBlock(0)} size="large">
-                                                                <Paste />
-                                                            </IconButton>
+                                                            <RowActionsMenu>
+                                                                <RowActionsItem
+                                                                    icon={<Visible />}
+                                                                    disabled={selectedCount === 0}
+                                                                    onClick={() => handleToggleVisibilityOfAllSelectedBlocks(true)}
+                                                                >
+                                                                    <FormattedMessage
+                                                                        id="comet.blocks.list.action.visible"
+                                                                        defaultMessage="Make visible"
+                                                                    />
+                                                                </RowActionsItem>
+                                                                <RowActionsItem
+                                                                    icon={<Invisible />}
+                                                                    disabled={selectedCount === 0}
+                                                                    onClick={() => handleToggleVisibilityOfAllSelectedBlocks()}
+                                                                >
+                                                                    <FormattedMessage
+                                                                        id="comet.blocks.list.action.invisible"
+                                                                        defaultMessage="Make invisible"
+                                                                    />
+                                                                </RowActionsItem>
+                                                                <Separator sx={{ marginLeft: 1, marginRight: 1 }} />
```

Comment:
Separator is already a `styled`-component. Should we define the margin there? IMO mixing `sx` and `styled` isn't needed here.

Reply:
I changed it so we can merge this